<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jesus Rivas</author>
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Concretus</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Venezuela"/>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Bienvenido a &lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:700;&quot;&gt;Concretus&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Comienza a diseñar tus mezclas de concreto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_pyqt6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>200</width>
      <height>226</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/pyqt_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_main">
    <property name="locale">
     <locale language="Spanish" country="Venezuela"/>
    </property>
    <property name="title">
     <string>Inicio</string>
    </property>
    <widget class="QMenu" name="menu_settings">
     <property name="locale">
      <locale language="Spanish" country="Venezuela"/>
     </property>
     <property name="title">
      <string>Configuración</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
     </property>
     <widget class="QMenu" name="menu_units">
      <property name="locale">
       <locale language="Spanish" country="Venezuela"/>
      </property>
      <property name="title">
       <string>Unidades</string>
      </property>
      <addaction name="action_SI"/>
      <addaction name="action_MKS"/>
     </widget>
     <addaction name="menu_units"/>
    </widget>
    <widget class="QMenu" name="menu_report">
     <property name="locale">
      <locale language="Spanish" country="Venezuela"/>
     </property>
     <property name="title">
      <string>Imprimir reporte</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/print.png</normaloff>images/print.png</iconset>
     </property>
     <addaction name="action_basic_report"/>
     <addaction name="action_full_report"/>
    </widget>
    <addaction name="menu_settings"/>
    <addaction name="menu_report"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_method">
    <property name="locale">
     <locale language="Spanish" country="Venezuela"/>
    </property>
    <property name="title">
     <string>Método</string>
    </property>
    <addaction name="action_MCE"/>
    <addaction name="action_ACI"/>
    <addaction name="action_DoE"/>
   </widget>
   <widget class="QMenu" name="menu_design">
    <property name="locale">
     <locale language="Spanish" country="Venezuela"/>
    </property>
    <property name="title">
     <string>Diseño</string>
    </property>
    <addaction name="action_regular_concrete"/>
   </widget>
   <widget class="QMenu" name="menu_proportioning">
    <property name="locale">
     <locale language="Spanish" country="Venezuela"/>
    </property>
    <property name="title">
     <string>Dosificación</string>
    </property>
    <addaction name="action_design_verification"/>
    <addaction name="action_trial_mix"/>
    <addaction name="action_mixture_adjustments"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="locale">
     <locale language="Spanish" country="Venezuela"/>
    </property>
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_main"/>
   <addaction name="menu_method"/>
   <addaction name="menu_design"/>
   <addaction name="menu_proportioning"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="action_SI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sistema Internacional de Unidades</string>
   </property>
  </action>
  <action name="action_MKS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sistema Técnico de Unidades</string>
   </property>
  </action>
  <action name="action_basic_report">
   <property name="text">
    <string>Reporte básico</string>
   </property>
  </action>
  <action name="action_full_report">
   <property name="text">
    <string>Reporte completo</string>
   </property>
  </action>
  <action name="action_MCE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MCE</string>
   </property>
  </action>
  <action name="action_ACI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ACI</string>
   </property>
  </action>
  <action name="action_DoE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DoE</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>images/about.png</normaloff>images/about.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="action_regular_concrete">
   <property name="text">
    <string>Concreto normal</string>
   </property>
  </action>
  <action name="action_design_verification">
   <property name="icon">
    <iconset>
     <normaloff>images/check.png</normaloff>images/check.png</iconset>
   </property>
   <property name="text">
    <string>Verificación del diseño</string>
   </property>
  </action>
  <action name="action_trial_mix">
   <property name="icon">
    <iconset>
     <normaloff>images/cement_mixer.png</normaloff>images/cement_mixer.png</iconset>
   </property>
   <property name="text">
    <string>Mezcla de prueba</string>
   </property>
  </action>
  <action name="action_mixture_adjustments">
   <property name="icon">
    <iconset>
     <normaloff>images/Adjustment.png</normaloff>images/Adjustment.png</iconset>
   </property>
   <property name="text">
    <string>Ajustes de la mezcla</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
